Source/Core/VideoBackends/OGL/VertexManager.cpp

```diff
@@ -178,6 +180,21 @@ void VertexManager::vFlush()
 #endif
   g_Config.iSaveTargetId++;
 
+  if (!g_Config.backend_info.bSupportsFragmentStoresAndAtomics && ::BoundingBox::active)
```

Comment:
This should likely be extracted to its own helper function which is called within this one.